test/runtests.jl

```diff
@@ -183,6 +184,14 @@ function test_opt(m::M) where {M}
     end
 end
 
+## Test ForwardDiff
+x = ones(2)
+A = [1.0 2.0; 3.0 4.0]
+b = [2.0, 2.0]
+for Tret in (Const,), Tx in (Const, Duplicated), Tb in (Const, Duplicated)
+    test_forward(VLBISkyModels._jlnuft!, Tret, (x, Tx), (A, Const), (b, Tb))
+end
```

Comment:
This isn't quite right. You aren't testing the rule and the shapes don't quite line up. I just added proper tests for the reverse rule, so you should be able to just use that code.